source/block-reconciler.ts

```diff
@@ -1,95 +1,99 @@
-import { Block } from "./models/block";
-import { BlockHistory } from "./models/block-history";
-import { List as ImmutableList } from "immutable";
-
-type GetBlockByHash<TBlock> = (hash: string) => Promise<TBlock|null>;
-
-export const reconcileBlockHistory = async <TBlock extends Block>(
-	getBlockByHash: GetBlockByHash<TBlock>,
-	blockHistory: BlockHistory<TBlock>|Promise<BlockHistory<TBlock>>,
-	newBlock: TBlock,
-	onBlockAdded: (block: TBlock) => Promise<void>,
-	onBlockRemoved: (block: TBlock) => Promise<void>,
-	blockRetention: number = 100,
-): Promise<BlockHistory<TBlock>> => {
-	blockHistory = await blockHistory;
-	if (isFirstBlock(blockHistory))
-		return await addNewHeadBlock(blockHistory, newBlock, onBlockAdded, blockRetention);
+import { Block } from './models/block'
+import { BlockHistory } from './models/block-history'
+
+type GetBlockByHash<TBlock> = (hash: string) => Promise<TBlock | null>
+
+export const reconcileBlockHistory = async <TBlock extends Block>(getBlockByHash: GetBlockByHash<TBlock>, blockHistory: BlockHistory<TBlock> | Promise<BlockHistory<TBlock>>, newBlock: TBlock, onBlockAdded: (block: TBlock) => Promise<void>, onBlockRemoved: (block: TBlock) => Promise<void>, blockRetention: number = 100): Promise<BlockHistory<TBlock>> => {
```

Comment:
Does the tool convert what was here to this, (meaning it forces everything to one-line even if the author split it across multiple lines)?  I'll need to take some time to read back over this code to see if there was a reason I split a handful of these functions onto multiple lines or if I was just being annoyingly inconsistent in my style.  My concern is that there are occasions where multi-line is intentional (because for one reason or another that line should be read in its entirety and not "skipped over" like previously discussed).

Reply:
Yes, it is an auto-formatter.